qtkeychain/keychain_win.cpp

```diff
@@ -9,18 +9,91 @@
 #include "keychain_p.h"
 #include "plaintextstore_p.h"
 
+#include <comdef.h>
 #include <windows.h>
+#include <wincred.h>
 #include <wincrypt.h>
 
 #include <memory>
 
 using namespace QKeychain;
 
+namespace {
+    constexpr std::wstring_view PRODUCT_NAME = L"QtKeychain";
```

Comment:
Hmm, this would require C++17, can we do without views?

Reply:
sure